libs/openchallenges/org-profile/src/lib/org-profile-overview/org-profile-overview.component.html

```diff
@@ -22,8 +22,13 @@ <h3 class="section-title">Organization Details</h3>
       </tr>
     </table>
   </div>
-  <hr>
+  <hr />
   <span class="mat-caption created-updated-dates"
-    >Added to the OC on {{ org.createdAt }} and last modified on {{ org.updatedAt }}
+    >Added to the OC on {{ org.createdAt }} and last modified on {{ org.updatedAt }} //
+    See something missing or not up-to-date?
+    <a
+      href="https://docs.google.com/forms/d/e/1FAIpQLSfQbQl11MG_5TXT7fL07YenRD3MIQYfG2lXrpo3jnh_iA4nsw/viewform?usp=pp_url&entry.1040159972={{org.login}}"
```

Comment:
Same comment as above: I suggest to use the org ID. Note that the org login should definitively less mutable than the challenge slug. Yet there is still a chance that an org would change it, especially if they want one that is different from the one we picked for them for the initial record.